src/components/themed/TransactionListComponents.js

```diff
@@ -4,21 +4,22 @@ import * as React from 'react'
 import { ActivityIndicator, View } from 'react-native'
 import { connect } from 'react-redux'
 
+import { Gradient } from '../../modules/UI/components/Gradient/Gradient.ui.js'
 import { type RootState } from '../../types/reduxTypes.js'
 import { type Theme, cacheStyles, useTheme } from '../services/ThemeContext.js'
 import { EdgeText } from '../themed/EdgeText.js'
 import { TransactionListHeader } from '../themed/TransactionListHeader.js'
 
-function HeaderComponent(props: { loading: boolean }) {
-  return props.loading ? <ActivityIndicator style={{ flex: 1, alignSelf: 'center' }} size="large" /> : <TransactionListHeader />
+function HeaderComponent(props: { loading: boolean, isEmpty: boolean }) {
+  return props.loading ? <ActivityIndicator style={{ flex: 1, alignSelf: 'center' }} size="large" /> : <TransactionListHeader isEmpty={props.isEmpty} />
 }
 
 export function SectionHeader(props: { title?: string }) {
   const styles = getStyles(useTheme())
   return (
-    <View style={styles.headerContainer}>
+    <Gradient style={styles.headerContainer}>
```

Comment:
Why did you put a gradient here? Doesn’t seem right

Reply:
Instead, set the color to a sectionHeaderBackgroundColor which happens to be transparent

Reply:
Because if you make it transparent. The sticky headers will just be a text and you will see other text pass thru it